### PR TITLE
BE-test-coverage: Unit tests coverage

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/ReturnsConnectorSpec.scala [hunk 1]
```diff
@@ -192,6 +193,53 @@ class ReturnsConnectorSpec extends WordSpec with Matchers with MockFactory with
 
       }
 
+      "handling request to amend a return" must {
```
Comment (AndyHWChung) [inline test/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/ReturnsConnectorSpec.scala:196]: this endpoint was accidentally left in - we should delete the amend return endpoint in the connector. We only used it for testing - we won't use it in it's current form when we actually come to do amendments.

Comment (rambabu-posa) [reply test/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/ReturnsConnectorSpec.scala:196]: Do you want me remove 'amend' from connector too?

Comment (AndyHWChung) [reply test/uk/gov/hmrc/cgtpropertydisposals/connectors/returns/ReturnsConnectorSpec.scala:196]: yeah, that'd be best for now - that implementation isn't valid

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**